src/main/kotlin/com/jaredsburrows/spoon/SpoonTask.kt

```diff
@@ -113,6 +113,6 @@ open class SpoonTask : DefaultTask() {
   }
 
   private fun getClickableFileUrl(path: File, fileName: String): String {
-    return URI("file", "", File(path.toURI().path, fileName).path, null, null).toString()
```

Comment:
I got this code from Gradle itself: https://github.com/gradle/gradle/blob/f001402c8b7f404c923d972f1bc524b2de7b3fcd/subprojects/logging/src/main/java/org/gradle/internal/logging/ConsoleRenderer.java#L37.

Reply:
Indeed, after my change it does generate URLs like `file:/reports/index.html`.
On the other hand, that code does cause #31 .
I'll look into it.

Reply:
But are they clickable in the terminal/windows console?

Reply:
Bump

Reply:
Sorry, I forgot about this issue. In the meantime I changed my workstation, but I will set up a Windows machine and look into it. Will return in a few days with an answer.

Reply:
Sounds good.